concepticondata/conceptlists/Wilson-2015-173.tsv-metadata.json

```diff
@@ -0,0 +1,90 @@
+{
+    "@context": [
+        "http://www.w3.org/ns/csvw",
+        {
+            "@language": "en"
+        }
+    ],
+    "dialect": {
+        "encoding": "utf-8-sig",
+        "delimiter": "\t",
+        "skipBlankRows": true
+    },
+    "tables": [
+        {
+            "tableSchema": {
+                "columns": [
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[a-zA-Z]+\\-[0-9]{4}\\-[0-9]+[a-z]?\\-[0-9]+[a-z]?$"
+                        },
+                        "name": "ID"
+                    },
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[0-9\\.]+([a-z\\\u2013]+)?$"
+                        },
+                        "name": "NUMBER"
+                    },
+                    {
+                        "datatype": {
+                            "base": "integer",
+                            "minimum": "1"
+                        },
+                        "name": "CONCEPTICON_ID"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "CONCEPTICON_GLOSS"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "ENGLISH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "MEANING"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "LATIN"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "SPANISH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "SCORE"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "POS"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "NO_OF_STATES"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "MEAN_RATE"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "RATE_SD"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "MSP_FREQUENCY"
```

Comment:
@mathildavz Could you explain what the abbreviation MSP stands for?

Reply:
@AnnikaTjuka The acronym doesn't occur in the Master's thesis. However, I have taken a look at different occurrences of MSP in linguistic papers and it could stand for _mean size of paradigm_, _masculine sound plural_ or _multiple strategy perspective_. I doubt the last one, I think it might have to do something with inflection? But I'm not sure.

Reply:
Could it be something like "mean Spanish" or "Modern Spanish"? Would you be able to dig into this and find out what the measure stands for? This is important to interpret the values in the column.

Reply:
Of course! I'll do my best to look into this.

Reply:
@AnnikaTjuka I've read huge parts of the paper and, unfortunately, I couldn't find a clear answer. I think, however, that your idea that it means "Modern Spanish" is most likely, as the paper talks a lot about the frequencies in Modern Spanish.

Reply:
Ok, then we can use FREQUENCY as a header. I'll also show you what we do with columns like this since we can integrate it as a NoRaRe dataset.